src/testTree.ts

```diff
@@ -99,15 +89,20 @@ export class TestTree extends DisposableBase {
   }
 
   private _update() {
-    for (const [workspaceFolder, workspaceRootItem] of this._rootItems) {
+    for (const workspaceFolder of this._vscode.workspace.workspaceFolders ?? []) {
```

Comment:
I think the folder might be stuck if user removes a workspaceFolder.

Reply:
Addressed.